packages/components/scripts/vendorism/update-vscode-readonly-files.js

```diff
@@ -87,7 +91,15 @@ export async function updateVsCodeReadOnlyFiles(
     // Write the updated settings back to settings.json
     console.log('🖊️ Writing to settings.json file...');
     await fs.mkdirSync(settingsPath.split('/').slice(0, -1).join('/'), { recursive: true });
-    await fs.writeFileSync(settingsPath, `${JSON.stringify(settings, null, 4)}\n`, 'utf8');
+    await fs.writeFileSync(settingsPath, JSON.stringify(settings), 'utf8');
+
+    const blob = fs.readFileSync(settingsPath);
+    const formattedFile = await prettier.format(blob.toString(), {
+      ...prettierConfig,
+      parser: 'json',
+    });
+
+    await fs.writeFileSync(settingsPath, formattedFile, 'utf8');
```

Comment:
Unfortunately not, because this is coupled with the `jobs` api for builds. I could have created a generic function that does it, but in this case, you would also just have saved two lines of code versus the import and usage, so I opted to leave it alone in this case.